vendor/github.com/prometheus/prometheus/tsdb/db.go

```diff
@@ -983,9 +983,9 @@ type dbAppender struct {
 
 var _ storage.GetRef = dbAppender{}
 
-func (a dbAppender) GetRef(lset labels.Labels) (storage.SeriesRef, labels.Labels) {
+func (a dbAppender) GetRef(lset labels.Labels, hash uint64) (storage.SeriesRef, labels.Labels) {
```

Comment:
We probably need to update go mod to make CI happy

Reply:
I wonder the vendor always git download from github/prometheus...... So CI ignores the committed vendor files in this PR.
Prometheus PR is in pending 
https://github.com/prometheus/prometheus/pull/11485

Reply:
I see. So it is not merged yet. Let's wait then.

Reply:
https://github.com/prometheus/prometheus/pull/11485 had merged, let's keep going.

Reply:
Seems we need to update prometheus here, right?

Reply:
But Prometheus latest release（v2.40.5） hasn't inclouded this change yet, https://github.com/prometheus/prometheus/blob/v2.40.5/storage/interface.go#L251, can we use Prometheus `main` branch here?